debugtools/DDR_VM/src/com/ibm/j9ddr/view/dtfj/image/J9DDRImageProcess.java

```diff
@@ -592,6 +593,7 @@ private int resolveGenericSignal(int num) {
 		}
 		if ((num & J9PORT_SIG_FLAG_SIGBUS) != 0) 	return 10;
 		if ((num & J9PORT_SIG_FLAG_SIGSEGV) != 0) 	return 11;
+		if ((num & J9PORT_SIG_FLAG_SIGUSR2) != 0) 	return 12;
```

Comment:
These tests are incorrect: on some platforms the values have multiple bits which must all be set for a match.

Reply:
I think I see what you mean in `omr/include_core/omrport.h`:

```
#if defined(OMR_OS_WINDOWS) || defined(OMRZTPF)
[...]
#define OMRPORT_SIG_FLAG_SIGSEGV                 0x4
[...]
#else /* defined(OMR_OS_WINDOWS) || defined(OMRZTPF) */
[...]
#define OMRPORT_SIG_SMALLEST_SIGNAL_FLAG        ((uint32_t)0x10)
[...]
#define OMRPORT_SIG_FLAG_SIGSEGV                ((OMRPORT_SIG_SMALLEST_SIGNAL_FLAG <<  0) | OMRPORT_SIG_FLAG_IS_SYNC)
[...]
#endif /* defined(OMR_OS_WINDOWS) || defined(OMRZTPF) */
```

I guess this goes to the previous point of needing an OS-specific `IProcess.resolveGenericSignal`